validation/standard-config.go

```diff
@@ -0,0 +1,49 @@
+package validation
+
+import "math/big"
+
+// StandardConfig is keyed by superchain target, e.g. "mainnet" or "sepolia" or "sepolia-dev-0"
+var StandardConfig map[string]*StandardConfigTy
```

Comment:
Let's make a new issue to use a string enum type for this key and other places the superchain name is used.